ext/date/lib/tm2unixtime.c

```diff
@@ -334,9 +334,9 @@ static timelib_sll do_years(timelib_sll year)
 static timelib_sll do_months(timelib_ull month, timelib_ull year)
 {
 	if (timelib_is_leap(year)) {
-		return ((month_tab_leap[month - 1] + 1) * SECS_PER_DAY);
+		return (((timelib_ull)month_tab_leap[month - 1] + 1) * SECS_PER_DAY);
```

Comment:
There's no potential - just as if you multiply 2*2 there's no potential for overflow.